### ISSUE TITLE
No dark theme?

### ISSUE THREAD

**Issue by 3c71** [Applandeo/Material-Calendar-View#246]
Is there a way to set dark theme? I can't even change background to black, or any other colors...

**Comment by kormateusz** [comment]
https://github.com/Applandeo/Material-Calendar-View#customization

**Comment by 3c71** [comment]
Thanks for closing, but those customization concerns very few elements, and none of them seem to affect the calendar background showing in white. It's unusable when the app is using a dark theme.

**Comment by kormateusz** [comment]
Calendar pages color: app:pagesColor="[color]"

**Comment by 3c71** [comment]
As I want to use transparent, I tried to use that trick with pagesColor, abbreviationsBarColor set to transparent in XML and this:
CalendarView cv = vg.findViewById(R.id.mycalendar);
cv.setBackgroundResource( drop_shadow );

Transparent does not actually work and results in dark only.

I also had to set abbreviationsLabelsColor in XML, which I hoped to avoid.

I could set the remaining colors in code, thanks to the other trick with CalendarProperties. But the 3 above properties do not work/exist in code.

It results in a messy code so I extended the CalendarView class and using reflection with the CalendarProperties trick to make it work in all scenarios without XML, so that the widget can be used transparently on dark or light theme, with or without custom colors.

In any above workaround the horizontal separators remains invisible in dark theme.

Adding methods set/getProperties which would work with all properties would be a nice way to improve support for dark theme, but until then dark theme is not supported and requires workaround.
